miscellaneous/build_templates/config.yml

```diff
@@ -23,10 +23,10 @@ steps:
       #flattenFolders: false # Optional
     displayName: 'Copy library config file'
 
-  - bash: rm ${{ parameters.fileOriginal }}
+  - bash: rm "${{ parameters.fileOriginal }}"
```

Comment:
In this file as well the quotes shouldn't be necessary if we can replace the spaces in the folder name with underscores.